Test.Microsoft.NET.Build.Containers.Filesystem/LayerEndToEnd.cs

```diff
@@ -22,10 +23,14 @@ public void SingleFileInFolder()
         Console.WriteLine(l.Descriptor);
 
         //Assert.AreEqual("application/vnd.oci.image.layer.v1.tar", l.Descriptor.MediaType); // TODO: configurability
-        Assert.IsTrue(l.Descriptor.Size is >= 135 and <= 200, $"'l.Descriptor.Size' should be between 135 and 200, but is {l.Descriptor.Size}"); // TODO: determinism!
+        Assert.IsTrue(l.Descriptor.Size is >= 135 and <= 300, $"'l.Descriptor.Size' should be between 135 and 300, but is {l.Descriptor.Size}"); // TODO: determinism!
```

Comment:
Where is this size difference coming from? Are PAX entries larger in size, generally?

Reply:
It looks to me that generally the tar sizes are known to be non-deterministic (likely due to gzip compression?). The key differences in size generally come from: 

1. Additional tar entries for the directories
2. Extended Attributes being written as separate entries in the PAX Format (e.g. for info like permissions)

I will increase the size range of the test like on the other test. Locally various test runs were green so I thought 300 is enough 😅